Model/GetVersion.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+
+namespace Bold\Checkout\Model;
+
+use Bold\Checkout\Api\GetVersionInterface;
+
+class GetVersion implements GetVersionInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function getVersion(string $shopId): int {
+        return 1;
```

Comment:
Should we be storing this version somewhere? Should we store this version as a string instead and then we can return a semantic version string?

Reply:
Maybe we can manage version in composer.json file and get it like:
```
$composerJsonPath = 'path/to/composer.json';
$jsonContents = file_get_contents($composerJsonPath);
$composerData = json_decode($jsonContents, true);
$version = $composerData['version'];
```

in this case, we will need to manage version of the module in one place during new release creation

Reply:
Either way it would be nice to use the same versioning that we will be using in composer. I don't think it would be great to have to maintain to version number systems, 1 for composer releases and one internally for the platform connector.